private/prepare_freq_matrices.m

```diff
@@ -95,9 +95,9 @@
 % update the cfg
 cfg.channel     = freq.label(chanindx);
 if any(strncmp(tok, 'rpt', 3))
```

Comment:
rather than the `any(strncmp(...))` I think this would better be explicitly tested as `startsWith(dimord, 'rpt')`